src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.service;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionService {
+    MovieSession add(MovieSession movieSession);
+
+    MovieSession get(Long id);
```

Comment:
Refer to the common mistakes:
<img width="664" alt="image" src="https://github.com/mate-academy/hibernate-create-movie-session-hw/assets/140437083/65006c51-8fba-40e8-89c4-0323a0aaef90">